tests/utils_spec.lua

```diff
@@ -40,3 +40,20 @@ describe('utils.non_empty', function()
     assert.are.same(true, result)
   end)
 end)
+
+describe('utils.fmt_error', function ()
+  it('interpolates message objects with variables', function ()
+    assert.are.equal('Hello, John!', require('dap.utils').fmt_error('{greeting}, {name}!', {
+      greeting = 'Hello',
+      name = 'John',
+    }))
+
+    assert.are.equal('Hello, -!', require('dap.utils').fmt_error('{greeting}, {unbound}!', {
+      greeting = 'Hello',
+    }))
+  end)
+
+  it('interpolates message objects without variables', function ()
+    assert.are.equal('Hello, John!', require('dap.utils').fmt_error('Hello, John!'))
+  end)
+end)
```

Comment:
I added my own implementation for this first and didn't see one already existed, figured I'd keep the tests